src/main/webapp/organization.js

```diff
@@ -384,7 +381,11 @@ class Organization {
   }
 
   convertIdsToEmails(moderators) {
```

Comment:
Add a comment that this takes a list of moderators and returns a single comma separated string. When i first read the function name i thought it would return a list as well.